__tests__/app.test.js

```diff
@@ -12,25 +14,40 @@ beforeEach(() => {
   return seed(data);
 });
 
-describe('/api/topics', () => {
-  test('GET: 200 Get all topics', () => {
-    return request(app).get("/api/topics")
-    .expect(200)
-    .then(({body}) => {
-      const {topics} = body;
+describe("/api/topics", () => {
+  test("GET: 200 Get all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
 
-      expect(Array.isArray(topics)).toBe(true)
-      topics.forEach((topic) => {
-        expect(typeof topic.slug).toBe('string')
-        expect(typeof topic.description).toBe('string')
-      })
+        expect(Array.isArray(topics)).toBe(true);
+        topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
 
-      expect(topics).toHaveLength(3)
-    });
-  })
-  test('GET: 404 When given in invalid path', () => {
-      return request(app).get('/api/banana').expect(404).then(({body}) => {
-        expect(body.msg).toBe('path doesn\'t exist')
-      })
-  })
-})
\ No newline at end of file
+        expect(topics).toHaveLength(3);
+      });
+  });
+  test("GET: 404 When given in invalid path", () => {
```

Comment:
This 404 is not specific to this endpoint, it can be in a separate describe block 👍